### PR TITLE
Fix docs: Broken links etc

### PR DESCRIPTION
**Pull request by kumare3**

### CODE REVIEW

File: rsts/faq/gcp.rst [hunk 1]
```diff
@@ -7,10 +7,15 @@ Google Cloud Platform
...
 -------------------------------------------------------
  Steps:
- - Are you using Workload Identity, then you have to pass in the ServiceAccount when you create the launchplan. Docs here https://lyft.github.io/flyte/user/features/roles.html?highlight=serviceaccount#kubernetes-serviceaccount-examples More information about WorkloadIdentity at https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
- - If you are just using a simple Nodepool wide permissions then check the cluster's ServiceACcount for Storage permissions. Do they look fine?
 
- - If not, then start a dummy pod in the intended namespace and check for ::
+ #. Are you using Workload Identity, then you have to pass in the ServiceAccount when you create the launchplan.
+     - Refer to docs :ref:`howto-serviceaccounts`
+     - More information about WorkloadIdentity at https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+ #. If you are just using a simple Nodepool wide permissions then check the cluster's ServiceACcount for Storage permissions. Do they look fine?
```
Comment (katrogan) [inline rsts/faq/gcp.rst:14]:
```suggestion
 #. If you are just using a simple Nodepool wide permissions then check the cluster's ServiceAccount for Storage permissions. Do they look fine?
```

File: rsts/howto/flytecli.rst [hunk 2]
```diff
@@ -38,7 +38,7 @@ the root directory of the flytecli package, and install the dependencies with
 ``pip install -e .``.
 
 
-.. _doc: https://virtualenv.pypa.io/en/latest/installation/
+.. _doc: https://virtualenv.pypi.io/en/latest/installation/
```
Comment (katrogan) [inline rsts/howto/flytecli.rst:41]: this link doesn't work for me either

Comment (wild-endeavor) [inline rsts/howto/flytecli.rst:41]:
```suggestion
.. _doc: https://virtualenv.pypa.io/en/latest/installation.html
```

Comment (wild-endeavor) [reply rsts/howto/flytecli.rst:41]: suggested.

**Review by wild-endeavor — APPROVED** [review]

**Review by katrogan — APPROVED** [review]

**Pull request merged**